tests/integrated/test-multigrid_laplace/runtest

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+#
+# Run the test, check the error
+#
+
+from __future__ import print_function
+try:
+    from builtins import str
+except:
+    pass
+
+tol = 1e-9 # Absolute tolerance
+numTests = 4 # We test 4 different boundary conditions (with slightly different inputs for each)
+
+from boututils.run_wrapper import shell, shell_safe, launch_safe, getmpirun
+from boutdata.collect import collect
+from sys import exit
+
+MPIRUN = getmpirun()
+
+print("Making multigrid Laplacian inversion test")
+shell("rm test_multigrid_laplace")
+shell_safe("make > make.log")
+
+print("Running multigrid Laplacian inversion test")
+success = True
+
+for nproc in [1,2,4]:
```

Comment:
On travis this will be slow, we get max ~ 2 cores, so 4 mpi processes is already oversubscription. Combining with two omp threads this will slow down significantly more (I suspect).